### PR TITLE
Feature register and login user

### LINKED ISSUE #1: User must login in the App
- Description: 
**As a** user
**I must** login in the App using an email and a password, 
**so that** I can use the App.

- Error State: 
**If** the email or password of an user is incorrect, or the email doesn’t exist, 
**then** show a message: “Your email or password is invalid. Please, try again”.

<div align="center">
Login - User
</div>

### PR DESCRIPTION
**Pull request by liannivin**
### Status 

Required

### Implementation Details
- Add Login and Signup components.
- Add styles for Login and Signup components.
- Add API for Login and Signup components.

close #1 #2

### CODE REVIEW

**Comment by cristianbgp** [comment]
@liannivin Please add a description to this PR, also remember to add the issues that this PR is closing

File: api/app/controllers/sessions_controller.rb [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+      render json: user
+    else
+      render json: { errors: 'Incorrect email or password' },
+             status: :bad_request
+    end
+  end
+
+  def register
+    user = User.new(user_params)
+    puts user
```
Comment (cristianbgp) [inline api/app/controllers/sessions_controller.rb]: I'd be nice to delete all the `puts`

**Comment by condef5** [comment]
Good Job, Lian `:)`

File: api/app/controllers/sessions_controller.rb (hunk 1 above)
Comment (liannivin) [reply api/app/controllers/sessions_controller.rb]: I deleted it. Thanks for this.

**Pull request merged**